assesspy/load_data.py

```diff
@@ -1,29 +1,52 @@
-# Import necessary libraries
+from importlib.resources import as_file, files
+
 import pandas as pd
-import pkg_resources
 
 
-# Load pre-made ratios sample data.
-def ratios_sample():
+def ccao_sample() -> pd.DataFrame:
     """
-    This sample was take from Evanston and New Trier in 2019. Ratios are
-    calculated using assessor certified (post-appeal) fair market values.
+    Sample of sales and estimated market values taken from Evanston and New
+    Trier in 2019. Estimates are Assessor certified (post-appeal) fair market
+    values.
 
     :return:
-        A data frame with 979 observation and 4 variables:
+        A Pandas DataFrame with 979 observation and 3 variables:
 
-        ======================== =======================================================
-        **assessed** (`float`)   The fair market assessed value predicted by CCAO
-                                 assessment models, including any successful appeals
-        **sale_price** (`float`) The recorded sale price of this property
-        **ratio** (`float`)      Sales ratio representing fair market value / sale price
-        **town** (`object`)      Township name the property is in
-        ======================== =======================================================
+        ============================== ============================================
+        **estimate** (`float`)         Fair market value predicted by CCAO
+                                       assessment models, after any successful appeals
+        **sale_price** (`float`)       Recorded sale price of this property
+        **township_name** (`object`)   Name of the township containing the property
+        ============================== ============================================
 
-    :rtype: DataFrame
+    :rtype: pd.DataFrame
     """
+    source = files("assesspy").joinpath("data/ccao_sample.parquet")
+    with as_file(source) as file:
+        return pd.read_parquet(file)
+
+
+def quintos_sample() -> pd.DataFrame:
```

Comment:
I figured it would be nice to include the Quintos data as an export to use in tests and demos.